src/Tribe/Ajax/Dropdown.php

```diff
@@ -95,13 +95,62 @@ public function search_terms( $search, $page, $args, $source ) {
 	}
 
 	/**
-	 * Sorts Hierarchically all the Terms for Select2
+	 * Search for Posts using Select2
+	 *
+	 * @since  TBD
+	 *
+	 * @param  string|array        $search   Search string from Select2.
+	 * @param  int                 $page     When we deal with pagination.
+	 * @param  array,string,mixed> $args     Arguments to pass to the query.
+	 * @param  string|int          $selected Selected item ID.
+	 *
+	 * @return array
+	 */
+	public function search_posts( $search, $page = 1, $args, $selected ) {
+		$posts_per_page = 10;
+		$data           = [];
+
+		if ( ! empty( $search ) ) {
+			if ( is_array( $search ) ) {
+				// Newer SelectWoo uses a new search format.
+				$args['s'] = $search['term']; // post?
+			} else {
+				// For older pieces that still use Select2 format.
+				$args['s'] = $search;
+			}
+		}
+
+		$args['posts_per_page'] = $posts_per_page;
+		$args['paged']          = $page;
```

Comment:
Since that will not be used at all, I would skip the caching of meta and taxonomy terms and make things a bit faster:

```suggestion
		$args['paged']          = $page;
		$args['update_post_meta_cache'] = false;
		$args['update_post_term_cache'] = false;
```